src/App.css

```diff
@@ -97,8 +96,7 @@ body {
     height: 100vh; /* Use vh as a fallback for browsers that do not support Custom Properties */
     height: calc(var(--vh, 1vh) * 100);
     max-width: 100%;
-    margin: 0 auto;
-    padding: 10px;
+    /* margin: 0 auto; */
```

Comment:
No need for commented out code in prod

Reply:
Of course, removed commented code on new commit.